src/main/java/org/apache/datasketches/memory/UnsafeUtil.java

```diff
@@ -157,6 +157,30 @@ static long getFieldOffset(final Class<?> c, final String fieldName) {
     }
   }
 
+  static long getArrayBaseOffset(final Class<?> c) {
```

Comment:
Please add a comment explaining the necessity for this method.

Reply:
Added a comment.